tests/integration-tests/tests/ad_integration/test_ad_integration/test_ad_integration/pcluster.config.yaml

```diff
@@ -3,7 +3,7 @@ Image:
 HeadNode:
   InstanceType: {{ instance }}
   Networking:
-    SubnetId: {{ public_directory_subnet }}
+    SubnetId: {{ public_subnet_id }}
```

Comment:
these same changes need to be made in the update config as well

Reply:
Done